miner/worker.go

```diff
@@ -1095,13 +1109,35 @@ func (w *worker) commitTransactions(txs *types.TransactionsByPriceAndNonce, coin
 func (w *worker) commitNewWork(interrupt *int32, noempty bool, timestamp int64) {
 	w.mu.RLock()
 	defer w.mu.RUnlock()
+
 	tstart := time.Now()
 	parent := w.chain.CurrentBlock()
 
 	if parent.Time() >= uint64(timestamp) {
 		timestamp = int64(parent.Time() + 1)
 	}
+
 	num := parent.Number()
+	// we are the megabundle worker
```

Comment:
Stray comment?

Reply:
no, I thought it was useful cause reader must keep in mind the overloaded context of the worker, and megabundle is special enough of a case (arguably)

Reply:
Ah ok, can you re-word it then?